samples/Mvc.Server/Startup.cs

```diff
@@ -135,7 +135,7 @@ public class Startup {
                     var hasher = new PasswordHasher<Application>();
 
                     context.Applications.Add(new Application {
-                        Id = "myClient",
+                        ApplicationId = "myClient",
```

Comment:
Having both `Id` and `ApplicationId` in the same `Application` entity is confusing. Renaming the second one would make clearer that `ClientId` is not a primary key.